MC/bin/o2dpg_sim_workflow.py

```diff
@@ -912,6 +912,17 @@ def addQCPerTF(taskName, needs, readerCommand, configFilePath, objectsFile=''):
                 configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/tofdigits.json',
                 objectsFile='tofDigitsQC.root')
 
+     addQCPerTF(taskName='tofft0PIDQC',
+                needs=[ITSTPCMATCHtask['name'], TOFRECOtask['name'], FT0RECOtask['name']],
+                readerCommand='o2-global-track-cluster-reader --track-types "ITS-TPC,TPC,ITS-TPC-TOF,TPC-TOF" --cluster-types FT0',
+                configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/pidft0tof.json',
+                objectsFile='tofft0PIDQC.root')
+     addQCPerTF(taskName='tofPIDQC',
+                needs=[ITSTPCMATCHtask['name'], TOFRECOtask['name']],
+                readerCommand='o2-global-track-cluster-reader --track-types "ITS-TPC,TPC,ITS-TPC-TOF,TPC-TOF" --cluster-types none',
+                configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/pidtof.json',
+                objectsFile='tofPIDQC.root')
```

Comment:
i think you can use the default `objectsFile` here, there is no need to be explicit about it.